gen/enum.go

```diff
@@ -134,7 +146,11 @@ func enum(g Generator, spec *compile.EnumSpec) error {
 				<end ->
 				}
 			<end ->
-			return []byte(<$strconv>.FormatInt(int64(<$v>), 10)), nil
+			<if checkErrOnUnknownEnum ->
+				return nil, errors.New(fmt.Sprintf("unknown value for enum %s", <$v>))
```

Comment:
`errors.New(fmt.Sprintf(..))` is redundant. Please use `fmt.Errorf(..)` instead.

staticcheck will usually catch this except it's part of the template. https://staticcheck.io/docs/checks#S1028